AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_OcclusionCullingPlaneAdded.py

```diff
@@ -0,0 +1,159 @@
+        occlusion_culling_plane_component = occlusion_culling_plane_entity.\
+            add_component(AtomComponentProperties.occlusion_culling_plane())
+        Report.critical_result(
+            Tests.occlusion_culling_plane_component_added,
+            occlusion_culling_plane_entity.has_component(AtomComponentProperties.occlusion_culling_plane()))
+
+        # 3. UNDO the entity creation and component addition.
+        # -> UNDO component addition.
+        general.undo()
+        # -> UNDO naming entity.
+        general.undo()
+        # -> UNDO selecting entity.
+        general.undo()
+        # -> UNDO entity creation.
+        general.undo()
+        general.idle_wait_frames(1)
+        Report.result(Tests.creation_undo, not occlusion_culling_plane_entity.exists())
+
+        # 4. REDO the entity creation and component addition.
+        # -> REDO entity creation.
+        general.redo()
+        # -> REDO selecting entity.
+        general.redo()
+        # -> REDO naming entity.
+        general.redo()
+        # -> REDO component addition.
+        general.redo()
+        general.idle_wait_frames(1)
+        Report.result(Tests.creation_redo, occlusion_culling_plane_entity.exists())
+
+        # 5. Enter/Exit game mode.
+        TestHelper.enter_game_mode(Tests.enter_game_mode)
+        general.idle_wait_frames(1)
+        TestHelper.exit_game_mode(Tests.exit_game_mode)
+
+        # 6. Test IsHidden.
+        occlusion_culling_plane_entity.set_visibility_state(False)
+        Report.result(Tests.is_hidden, occlusion_culling_plane_entity.is_hidden() is True)
+
+        # 7. Test IsVisible.
+        occlusion_culling_plane_entity.set_visibility_state(True)
+        general.idle_wait_frames(1)
+        Report.result(Tests.is_visible, occlusion_culling_plane_entity.is_visible() is True)
+
+        # 8. Delete occlusion_culling_plane entity.
+        occlusion_culling_plane_entity.delete()
+        Report.result(Tests.entity_deleted, not occlusion_culling_plane_entity.exists())
+
+        # 9. UNDO deletion.
+        general.undo()
+        Report.result(Tests.deletion_undo, occlusion_culling_plane_entity.exists())
+
+        # 10. REDO deletion.
+        general.redo()
+        Report.result(Tests.deletion_redo, not occlusion_culling_plane_entity.exists())
+
+        # 11. Look for errors or asserts.
+        TestHelper.wait_for_condition(lambda: error_tracer.has_errors or error_tracer.has_asserts, 1.0)
+        for error_info in error_tracer.errors:
+            Report.info(f"Error: {error_info.filename} {error_info.function} | {error_info.message}")
+        for assert_info in error_tracer.asserts:
+            Report.info(f"Assert: {assert_info.filename} {assert_info.function} | {assert_info.message}")
+
+
+if __name__ == "__main__":
+    from editor_python_test_tools.utils import Report
+    Report.start_test(AtomEditorComponents_OcclusionCullingPlane_AddedToEntity)
```

Comment:
I don't see it in the PR but I assume there is a blank line at the end of the file correct? in the current version I'm reviewing it would be line 160 as a blank line after line 159 which contains `Report.start_test(AtomEditorComponents_OcclusionCullingPlane_AddedToEntity)`

Reply:
Line 160 is in there, and is blank